circle.yml

```diff
@@ -114,5 +157,65 @@ workflows:
   version: 2
   build-test-and-deploy:
     jobs:
-      - build
+      - build:
+          name: quality_migrations_tests
+          tests_group: 0
+          tests_type: clean test.quality test.migrations_missing
+      - build:
+          name: python_unit_tests
+          tests_group: 0
+          tests_type: clean test.unit
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: browser_tests
+          tests_group: 0
+          tests_type: clean test.browser
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: js_tests
+          tests_group: 0
+          tests_type: clean test.js
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: integration_tests_group_1
```

Comment:
@viadanna, afaik, we can have 4 shards in a run. So it might be a good idea to move the integration tests that don't spawn a full-fledged appserver to a separate shard. This also makes debugging SSL issues and other easier because all the other shards will have just one test that spawns a full-fledged appserver.

Reply:
:+1: